jina/docker/hubio.py

```diff
@@ -66,14 +67,52 @@ def new(self):
         elif self.args.type == 'pod':
             cookiecutter_template = 'https://github.com/jina-ai/cookiecutter-jina-hub.git'
         cookiecutter(cookiecutter_template, overwrite_if_exists=self.args.overwrite, output_dir=self.args.output_dir)
-        
+
         try:
-            cookiecutter(cookiecutter_template, overwrite_if_exists=self.args.overwrite, output_dir=self.args.output_dir)
+            cookiecutter(cookiecutter_template, overwrite_if_exists=self.args.overwrite,
+                         output_dir=self.args.output_dir)
         except click.exceptions.Abort:
             self.logger.info('nothing is created, bye!')
 
     def push(self, name: str = None, readme_path: str = None):
-        """A wrapper of docker push """
+        """ A wrapper of docker push """
+        is_build_success, is_push_success = True, False
+        try:
+            self._push_docker_hub(name, readme_path)
+            is_push_success = True
+        except:
+            self.logger.error(f'can not push to the registry')
+
+        file_path = get_summary_path(name)
+        if os.path.isfile(file_path):
+            with open(file_path) as f:
+                result = json.load(f)
```

Comment:
Given our discussion in Slack, the order here has to be changed.

1. check if we have "certificate" aka that temp file, if yes, continue, otherwise, log.error and return
2. push to docker
3. write to mongodb